public/app/features/explore/ExploreQueryInspector.tsx

```diff
@@ -56,6 +56,7 @@ export function ExploreQueryInspector(props: Props) {
     content: (
       <InspectDataTab
         data={dataFrames}
+        dataName={'Explore'}
```

Comment:
This to maintain the old logic of `panel ? panel.getDisplayTitle() : 'Explore'` that existed in a few places in InspectDataTab